easybuild/easyconfigs/i/Ipopt/Ipopt-3.12.9-foss-2017b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'Ipopt'
+version = '3.12.9'
+
+homepage = 'https://projects.coin-or.org/Ipopt'
+description = """ IPOPT (Interior Point Optimizer, pronounced Eye-Pea-Opt)
+ is an open source software package for large-scale nonlinear optimization. """
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['https://www.coin-or.org/download/source/Ipopt/']
+sources = [SOURCE_TGZ]
+checksums = ['8ff3fe1a8560896fc5559839a87c2530cac4ed231b0806e487bfd3cf2d294ab8']
+
+dependencies = [
+    ('AMPL-MP', '3.1.0'),
+]
+
+configopts = '--with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+configopts += ' --without-hsl'
+configopts += ' --with-asl-lib=-lasl --with-asl-incdir=$EBROOTAMPLMINMP/include/asl'
+
+sanity_check_paths = {
+    'files': ['bin/ipopt', 'lib/libipopt.so'],
```

Comment:
Please don't hardcode `.so`, use `SHLIB_EXT` instead please (`.so` is specific to Linux, we want to leave the door open for macOS too where a `.so` is a `.dylib`)

Maybe also check for `libipoptamplinterface.so`?

```python
'files': ['bin/ipopt', 'lib/libipopt.%s' % SHLIB_EXT, 'lib/libipoptamplinterface.%s' % SHLIB_EXT],